package.json

```diff
@@ -4,7 +4,7 @@
   "description": "React component library for Catch&Release",
   "main": "dist/index.js",
   "scripts": {
-    "test": "jest",
+    "test": "TZ=UTC jest",
```

Comment:
This is the root cause of time-based fixtures failing between hosts. It applies whatever TZ is on the host both when generating fixtures, and running tests. This env var resolves that, but I'm not sure if there is a better way to be passing this.